### ISSUE TITLE
ImportError: No module named mvpa2.suite

### ISSUE THREAD

**Issue by MunibaFaiza** [PyMVPA/PyMVPA#629]
I am unable to install mvpa2 suite on Ubuntu 20.04. I have tried sudo apt-get install python-mvpa2. But there is no repository for this. Same goes for the pip install. Please tell me how can I install it.

Thank you!

**Comment by anthonyromyn** [comment]
Hey @MunibaFaiza,

I was having similar trouble after moving to 20.04 but was able to get everything working nicely through conda following this: https://github.com/PyMVPA/PyMVPA/issues/624#issue-901162525.

Hope it's works for you!
Anthony

**Comment by yarikoptic** [comment]
Thanks @anthonyromyn . As long as you are still using python2 -- should indeed be good to go.

**Comment by MunibaFaiza** [comment]
Yes, @anthonyromyn thank you. I installed it successfully. I had to use Python2.
